.github/workflows/config.yml

```diff
@@ -0,0 +1,50 @@
+# Do not modify!
+# This file was generated from a template using https://github.com/StefMa/pkl-gha
+
+name: CI
+'on':
+  pull_request:
+    branches:
+    - main
+  push:
+    branches:
+    - main
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v5
+    - name: ''
```

Comment:
I don't think having an empty string name is a great idea, even if the syntax allows it.